popup.js

```diff
@@ -1,5 +1,7 @@
 document.addEventListener('DOMContentLoaded', function() {
-  document.getElementById('button').addEventListener('click', click);
+  if (document.getElementById('button')) {
```

Comment:
I added it because otherwise it throws an error when I import popup.js into my tests.